### PR TITLE
Darkmode Icon replacement and Color Enhancement for keywords

### PR DESCRIPTION
**Pull request by Baskercarrer**
* Added chocolate color for all gherkin keywords (Background, Scenario, Given, When, Then).

**Before** 
image

After Enhanced color for keywords

image

### CODE REVIEW

File: templates/style.css [hunk 1]
```diff
@@ -525,3 +525,15 @@ body.darkmode div pre {
 body.darkmode span.tag {
     color: #FFD119;
 }
+
+body.darkmode .keyword {
+    color: chocolate;
```
Comment (wswebcreation) [inline templates/style.css:530]: Thanks, but can we provide hex-color codes here instead of strings?

Comment (Baskercarrer) [reply templates/style.css]: Have updated with color code and also replace toggle buttons with moon/sun icon.

Comment (Baskercarrer) [reply templates/style.css]: Light mode
image

Darkmode
image

**Comment by Baskercarrer** [comment]
Please dont merge this PR at this moment. will update here about the state. I am planning to change the darkmode icon

**Comment by Baskercarrer** [comment]
Am done with change in darkmode toggle button. please feel free to review it.

**Comment by wswebcreation** [comment]
Thanks!

**Pull request merged**